blazor/dropdown-tree/templates.md

```diff
@@ -62,6 +62,73 @@ The variable `context` holds the data of the current node.
 
+## Value template
+
+The currently selected value that is displayed by default on the Dropdown Tree input element can be customized using the `ValueTemplate` property.
+
+In the following sample, the selected value is displayed as a combined text of both `Name` and `Job` in the Dropdown Tree input, which is separated by a hyphen.
+
+The template expression in Blazor should be provided using the `@` syntax directly inside the `ValueTemplate`. This allows the properties of the bound object to be rendered dynamically in the template.
+
+The variable `context` holds the data of the current node.
+
+```cshtml
+@using Syncfusion.Blazor.Navigations
+
+<SfDropDownTree TItem="EmployeeData" TValue="string" Placeholder="Select an employee" Width="500px">
+    <ChildContent>
+        <DropDownTreeField TItem="EmployeeData" DataSource="Data" ID="Id"  Text="Name" HasChildren="HasChild" ParentID="PId" Expanded="Expanded"></DropDownTreeField>
+    </ChildContent>
+    <ValueTemplate>
+        <span>@((context as EmployeeData).Name) - @((context as EmployeeData).Job)</span>
+    </ValueTemplate>
+</SfDropDownTree>
+
+@code {
+    List<EmployeeData> Data = new List<EmployeeData>
+    {
+        new EmployeeData() { Id = "1", Name = "Steven Buchanan", Job = "General Manager", HasChild = true, Expanded = true },
+        new EmployeeData() { Id = "2", PId = "1", Name = "Laura Callahan", Job = "Product Manager", HasChild = true },
+        new EmployeeData() { Id = "3", PId = "2", Name = "Andrew Fuller", Job = "Team Lead", HasChild = true },
+        new EmployeeData() { Id = "4", PId = "3", Name = "Anne Dodsworth", Job = "Developer" },
+        new EmployeeData() { Id = "10", PId = "3", Name = "Lilly", Job = "Developer" },
+        new EmployeeData() { Id = "5", PId = "1", Name = "Nancy Davolio", Job = "Product Manager", HasChild = true },
+        new EmployeeData() { Id = "6", PId = "5", Name = "Michael Suyama", Job = "Team Lead", HasChild = true },
+        new EmployeeData() { Id = "7", PId = "6", Name = "Robert King", Job = "Developer" },
+        new EmployeeData() { Id = "11", PId = "6", Name = "Mary", Job = "Developer" },
+        new EmployeeData() { Id = "9", PId = "1", Name = "Janet Leverling", Job = "HR"}
+    };
+
+    class EmployeeData
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string Job { get; set; }
+        public bool HasChild { get; set; }
+        public bool Expanded { get; set; }
+        public string PId { get; set; }
+    }
+}
+<style>
+    .control-wrapper {
+        max-width: 300px;
+        padding: 20px 0px 0px;
+        margin: 0 auto;
+    }
+    .default-label {
+        font-size: 14px;
+        margin-bottom: 6px;
+    }
+    @@media (max-width: 900px) {
+        .control-wrapper {
+            padding: 30px 0px 0px;
+        }
+    }
+</style>
+```
+
+![Blazor Dropdown Tree with ValueTemplate.](./images/template/blazor-dropdowntree-value-template.png)
```

Comment:
@Saravanan-sf4375 - Please add image with clarity and full view

Reply:
modified the image and updated the PR.